.dev/dev_arm64.yaml

```diff
@@ -1,132 +0,0 @@
-# This is a compose file designed for arm64/Apple Silicon systems
-# To adapt this to x86 please find and replace ".arm64" with empty
-
-# ARM64 supported images for kafka can be found here
-# https://hub.docker.com/r/confluentinc/cp-kafka/tags?page=1&name=arm64
----
-version: '3.8'
-name: "kafbat-ui-dev"
```

Comment:
Let's keep this compose please. It's a generic compose for dev purposes without any specific stuff other compose files provide (sasl, proxy, etc). We can remove the 'arm64' postfix in image names and rename the file instead.

Reply:
I updated it.